front/app/containers/Admin/users/ChangeSeatModal.tsx

```diff
@@ -152,8 +158,11 @@ const ChangeSeatModal = ({
               width="auto"
               onClick={() => {
                 changeRoles(userToChangeSeat, isChangingToNormalUser);
-                // We are only showing the success modal when changing to admin so we check if the user is not currently an admin
-                if (!isUserToChangeSeatAdmin) {
+                // We are only showing the success modal when a seat has been added
+                const isSeatBeingAdded =
+                  !isUserToChangeSeatAdmin &&
+                  !isChangingCollaboratorToNormalUser;
```

Comment:
This logic is still not super clear to me. Can we try to improve it?

So how I currently read this code:
1. If this modal appears, we're sure that we're changing some user role(s)
2. If the user to change is not an admin and the user to change is not going from mod to normal user, we show the success modal. 
3. Do we currently mean "user is changing from admin to normal user" with isUserToChangeSeatAdmin here? So basically deducting: if the user is currently an admin, we're setting them to a normal user here. If yes, can we make that more explicit? (If no, shouldn't we include that?)

Reply:
If my understanding is right, we can perhaps simplify this logic to 
`const isSeatBeingAdded = !isChangingToNormalUser;
`?
(since we're changing a seat/role for sure, and if it's not to a normal user, we're upgrading someone).

Reply:
The key thing to note here is that we are showing success when a seat is added. That is where the emphasis is. From that basis, a seat is bought when this component is displayed, the user (passed prop) is not an admin (changing to admin) and not a collaborator being made a normal user.

Reply:
Indeed, this could work 
```
const isSeatBeingAdded = !isChangingToNormalUser;
```

Reply:
Let's pick that? :-)